### PR TITLE
add_admin_pages

### PR DESCRIPTION
**Pull request by as-sergey-b**

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    div class: "blank_slate_container", id: "dashboard_default_message" do
+      span class: "blank_slate" do
+        span I18n.t("active_admin.dashboard_welcome.welcome")
+        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (IlyaUmanets) [inline app/admin/dashboard.rb]: Remove commented lines

File: app/assets/stylesheets/active_admin.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// SASS variable overrides must be declared before loading up Active Admin's styles.
```
Comment (IlyaUmanets) [inline app/assets/stylesheets/active_admin.scss]: Remove commented lines

File: app/models/character.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class Character < ApplicationRecord
+  acts_as_paranoid
```
Comment (IlyaUmanets) [inline app/models/character.rb]: - constants
- modules
- associations
- validations
- delegations
- gem methods
- class methods
- instance methods

So `acts_as_paranoid` must be near `enum`

File: app/models/user.rb [hunk 4]
```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
   has_secure_password
+  acts_as_paranoid
```
Comment (IlyaUmanets) [inline app/models/user.rb]: Same

File: config/initializers/active_admin.rb [hunk 5]
```diff
@@ -0,0 +1,287 @@
+ActiveAdmin.setup do |config|
+  # == Site Title
```
Comment (IlyaUmanets) [inline config/initializers/active_admin.rb]: Remove commented lines

File: db/migrate/20170227215658_create_active_admin_comments.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (IlyaUmanets) [inline db/migrate/20170227215658_create_active_admin_comments.rb]: Empty lines

File: app/admin/dashboard.rb (hunk 1 above)
Comment (SBlazhko) [inline app/admin/dashboard.rb]: fixed

File: app/assets/stylesheets/active_admin.scss (hunk 2 above)
Comment (SBlazhko) [inline app/assets/stylesheets/active_admin.scss]: fixed

File: app/models/character.rb (hunk 3 above)
Comment (SBlazhko) [inline app/models/character.rb]: fixed

File: app/models/user.rb (hunk 4 above)
Comment (SBlazhko) [inline app/models/user.rb]: fixed

File: db/migrate/20170227215658_create_active_admin_comments.rb (hunk 6 above)
Comment (SBlazhko) [inline db/migrate/20170227215658_create_active_admin_comments.rb]: fixed

File: config/initializers/active_admin.rb (hunk 5 above)
Comment (SBlazhko) [inline config/initializers/active_admin.rb]: fixed

**Pull request merged**